src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -29,4 +35,34 @@ class OppgaveController(val oppgaveService: OppgaveService) {
         return oppgaveService.getOppgaver()
     }
 
+    @GetMapping("/oppgaver/ikketildelt")
```

Comment:
Burde dette vært:

`/oppgaver?assigned=false` f.eks.?

Reply:
Nå blir jo "vi" eneste konsumenten av dette apiet, så vi kan jo være så pragmatiske som vi vil da. Jeg er heller ikke en ekspert på rest-api-design.. Vi kan jo bli enige i et utviklermøte om hvordan vi gjør det.

Reply:
Jeg pleier å være sånn passe pragmatisk når jeg lager slike endepunkt, og tenker at begge deler er greit, og at vi bør velge det som er best for frontend hvis de har en preferanse.

Reply:
Men er den miksen av norsk og engelsk ment å være med ut i URLene også altså?

Reply:
Hadde skrevet en lengre tekst her, men blir jo bare mine synspunkter. Vi i teamet må snakke sammen og bli enige om både rest-standard og språkbruk. F.eks. i forbindelse med neste morgenmøte.